exonum/src/explorer/mod.rs

```diff
@@ -294,32 +299,32 @@ impl BlockWithTransactions {
     }
 
     /// Iterates over transactions in the block.
-    pub fn iter(&self) -> EagerTransactions {
+    pub fn iter(&self) -> EagerTransactions<T> {
         self.transactions.iter()
     }
 }
 
 /// Iterator over transactions in [`BlockWithTransactions`].
 ///
 /// [`BlockWithTransactions`]: struct.BlockWithTransactions.html
-pub type EagerTransactions<'a> = ::std::slice::Iter<'a, CommittedTransaction>;
+pub type EagerTransactions<'a, T> = ::std::slice::Iter<'a, CommittedTransaction<T>>;
```

Comment:
I don't like paths starting with `::`. :upside_down_face: Would you mind adding `use std::slice`?